build.gradle

```diff
@@ -15,8 +15,11 @@ repositories {
 }
 
 dependencies {
+//	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
```

Comment:
If you have any unused dependencies i would just delete them before merging it to main.

Reply:
Oops, I did it again!